### PR TITLE
net-libs/tox and net im/toxic

### PR DESCRIPTION
**Pull request by xWNgLWT0mQf**

### CODE REVIEW

File: net-libs/tox/tox-9999.ebuild [hunk 1]
```diff
@@ -44,19 +45,18 @@ src_configure() {
...
 		-DBUILD_TOXAV=$(usex av)
 		-DDHT_BOOTSTRAP=$(usex dht-node)
 		-DENABLE_SHARED=ON
 		-DENABLE_STATIC=$(usex static-libs)
 		-DMUST_BUILD_TOXAV=$(usex av))
 	if use test; then
 		mycmakeargs+=(
-			-DBUILD_AV_TEST=$(usex av)
 			-DTEST_TIMEOUT_SECONDS=120
 			-DUSE_IPV6=$(usex ipv6))
```
Comment (thesamesam) [inline net-libs/tox/tox-9999.ebuild:57]: You only want IPv6 if tests?

File: net-libs/tox/tox-9999.ebuild [hunk 2]
```diff
@@ -97,15 +101,12 @@ pkg_postinst() {
...
-	if use daemon; then
-		if [[ -f ${EROOT}/var/lib/tox-dht-bootstrap/key ]]; then
-			ewarn "Backwards compatability with the bootstrap daemon might have been"
-			ewarn "broken a while ago. To resolve this issue, REMOVE the following files:"
-			ewarn "    ${EROOT}/var/lib/tox-dht-bootstrap/key"
-			ewarn "    ${EROOT}/etc/tox-bootstrapd.conf"
-			ewarn "    ${EROOT}/run/tox-dht-bootstrap/tox-dht-bootstrap.pid"
-			ewarn "Then just re-emerge net-libs/tox"
-		fi
+		ewarn ""
```
Comment (thesamesam) [inline net-libs/tox/tox-9999.ebuild:104]: Have these issues been reported upstream? Could you provide a bug reference?

File: net-libs/tox/tox-9999.ebuild [hunk 3]
```diff
@@ -19,23 +19,24 @@ RESTRICT="!test? ( test )"
...
+	daemon? (
+		acct-group/tox
+		acct-user/tox
+	)"
 
 src_prepare() {
 	cmake_src_prepare
 	#remove faulty tests
-	for testname in bootstrap lan_discovery save_compatibility tcp_relay tox_many_tcp; do
+	for testname in lan_discovery save_compatibility; do
```
Comment (thesamesam) [inline net-libs/tox/tox-9999.ebuild:39]: Above this, add a line: ```local testname=```.

File: net-libs/tox/tox-9999.ebuild [hunk 4]
```diff
@@ -77,6 +77,10 @@ src_configure() {
 	cmake_src_configure
```
Comment (thesamesam) [inline net-libs/tox/tox-9999.ebuild:77]: The collection of log-* USE flags are quite complicated. Is this just for runtime logging, or build time?

Could we use an environment variable or something instead? Or allow users to use the EXTRA_ECONF equivalent for cmake (the name escapes me right now)?

File: net-libs/tox/tox-9999.ebuild (hunk 1 above)
Comment (xWNgLWT0mQf) [reply net-libs/tox/tox-9999.ebuild:57]: The ipv6 cmake option is only used to "Use IPv6 in tests". See https://github.com/TokTok/c-toxcore/blob/master/INSTALL.md if interested.

File: net-libs/tox/tox-9999.ebuild (hunk 2 above)
Comment (xWNgLWT0mQf) [reply net-libs/tox/tox-9999.ebuild:104]: Yes, I will include a bug reference.

File: net-libs/tox/tox-9999.ebuild (hunk 4 above)
Comment (xWNgLWT0mQf) [reply net-libs/tox/tox-9999.ebuild:77]: The documentation seems to suggest that this option enables or disables the logging at the code level by using macros. I don't recognize thee EXTRA_ECONF reference, but I would be open to hear more about it.

File: net-libs/tox/tox-9999.ebuild (hunk 3 above)
Comment (xWNgLWT0mQf) [reply net-libs/tox/tox-9999.ebuild:39]: I am not entirely certain what you mean. I will include one possible interpretation in my updated pull request.

File: net-libs/tox/tox-9999.ebuild (hunk 4 above)
Comment (thesamesam) [reply net-libs/tox/tox-9999.ebuild:78]: See make.conf(5) for information about EXTRA_ECONF, it is an environment variable for people to pass in options to econf. But I guess we can keep this for now to just get a newer version in.

I'm just not sure about keeping so many log use flags in future.

**Comment by thesamesam** [comment]
Done, thank you!

**Pull request closed without merging**